config/_default/menus/menus.en.yaml

```diff
@@ -2049,26 +2049,31 @@ main:
     parent: ci
     identifier: setup_pipelines
     weight: 2
-  - name: GitLab
-    url: continuous_integration/setup_pipelines/gitlab/
-    parent: setup_pipelines
-    identifier: ci_gitlab
-    weight: 201
-  - name: Jenkins
-    url: continuous_integration/setup_pipelines/jenkins/
-    parent: setup_pipelines
-    identifier: ci_jenkins
-    weight: 202
   - name: Buildkite
     url: continuous_integration/setup_pipelines/buildkite/
     parent: setup_pipelines
     identifier: ci_buildkite
-    weight: 203
+    weight: 201
```

Comment:
I'll have the same order here as our in-app onboarding:

1. GitHub
2. Jenkins
3. CircleCI
4. GitLab
5. Buildkite

Reply:
I don't mind either way but the rest of the team suggested alphabetical order so it be easier to find things.